src/components/EntryEditor/EntryEditor.js

```diff
@@ -53,9 +53,15 @@ class EntryEditor extends Component {
     const controlClassName = `${ styles.controlPane } ${ this.state.showEventBlocker && styles.blocker }`;
     const previewClassName = `${ styles.previewPane } ${ this.state.showEventBlocker && styles.blocker }`;
 
+    const disablePreview = collection.getIn(['editor', 'disable_preview'], false);
```

Comment:
Tying into my comments about the change of the setting name, I would change this to something like `collectionHasPreview` to make it more clear where the source of this value is, particularly since there's already `this.state.previewVisible`.

Additionally, if the setting name changes to `editor.preview`, this should default to `true`.